src/main/java/org/influxdb/impl/BatchProcessor.java

```diff
@@ -13,6 +13,7 @@
 import java.util.logging.Level;
 import java.util.logging.Logger;
 
+import org.apache.commons.collections4.keyvalue.MultiKey;
```

Comment:
@majst01  I also like less dependency and less code changes. So I will change the code. If nobody compain it isn't very strict, I think it is OK. WDTY?